### PR TITLE
Shelly Motion: enable/disable motion detection support.

### PR DESCRIPTION
**Pull request by mstefany**
Hi. I am missing some nice features for Shelly devices in Home Assistant, e.g. enabling/disabling Shelly Motion motion detection, setting motion/tamper sensitivity, or changing Shelly Gas Valve addon state. I would like to contribute those to aioshelly and Home Assistant Core. I created this small PR to get a feedback and see the whole process. Thank you.

### CODE REVIEW

File: aioshelly/block_device.py [hunk 1]
```diff
@@ -268,6 +268,17 @@ async def trigger_shelly_gas_unmute(self) -> None:
         """Trigger a Shelly Gas unmute action."""
         await self.http_request("get", "unmute")
 
+    async def enable_shelly_motion_detection(self, enable: bool) -> None:
```
Comment (thecode) [inline aioshelly/block_device.py:271]:
```suggestion
    async def set_shelly_motion_detection(self, enable: bool) -> None:
```
To be consistent with other set commands we have

File: aioshelly/block_device.py [hunk 2]
```diff
@@ -268,6 +268,17 @@ async def trigger_shelly_gas_unmute(self) -> None:
...
+        """Enable or disable Shelly Motion motion detection."""
+        params = {}
+
+        if enable:
+            params = {"motion_enable": "true"}
+        else:
+            params = {"motion_enable": "false"}
+
```
Comment (thecode) [inline aioshelly/block_device.py:279]:
```suggestion
        params = {"motion_enable": "true"} if enable else {"motion_enable": "false"}
```

**Review by thecode — CHANGES_REQUESTED** [review]

File: aioshelly/block_device.py (hunk 1 above)
Comment (mstefany) [reply aioshelly/block_device.py]: OK, changed, thank you.

File: aioshelly/block_device.py (hunk 2 above)
Comment (mstefany) [reply aioshelly/block_device.py]: I was not sure if you write it in long or short form. Thank you for suggestion, I changed it.

**Review by thecode — APPROVED** [review]

**Review by bieniu — APPROVED** [review]
LGTM 👍

**Pull request merged**